bindings/azure/eventhubs/eventhubs.go

```diff
@@ -198,10 +198,11 @@ func (a *AzureEventHubs) RegisterPartitionedEventProcessor(handler func(*binding
 		return err
 	}
 
-	callback := func(c context.Context, event *eventhub.Event) error {
+	callback := func(c context.Context, event *eventhub.Event, bind *bindings.InvokeResponse) error {
```

Comment:
Does this compile? The [eventhubs.Hub.Receive](https://pkg.go.dev/github.com/Azure/azure-event-hubs-go/v3#Hub.Receive) takes a callback of type `Handler func(ctx context.Context, event *Event) error`.

Also note that any changes to `ReadResponse` will also need to be reflected in `RegisterEventProcessor` as well.

Reply:
ping @tanvigour